example/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
```

Comment:
When I run `"npm run watch"` inside the example it fails with an error `"Reference error React not defined"`

I think we need to add another option ( `"jsx": "react"` ) to the compiler options :


```
  "compilerOptions": {
    "module": "commonjs",
    "jsx": "react"
  },
```


Found this solution here, but there might be other ways to fix it  :
https://github.com/parcel-bundler/parcel/issues/1199